src/credentials/vue/authForms/CredentialsForm.vue

```diff
@@ -0,0 +1,280 @@
+<template>
```

Comment:
A name like `manageCredentials.vue` (or whatever is more appropriate) would express the purpose of this form more clearly than `CredentialsForm.vue`